_schedule/tutorials/2018-10-14-13-30-t2-what-to-expect-when-you-re-expecting-a.md

```diff
@@ -6,6 +6,7 @@ abstract: We all know we should be testing our applications, but testing is hard
 accepted: true
 category: tutorial
 date: 2018-10-14 13:30:00 -0500
+end_date: 2018-10-14 17:00:00 -0500
```

Comment:
On all of these, is `-0500` correct? I thought Pacific was `-0700` or `-0800` depending on DST. It's been awhile, though

Reply:
It's a workaround for whatever Timezone Github's servers are in. Last year, we had everything as `-0700` and at some point, everything was off three hours. The only fix we could find is what you are seeing. 

It's weird.

Reply:
They seem to ignore `_config.yml` too <or> they may have another workaround since it's been four or five months. Not sure.